.changeset/five-comics-march.md

```diff
@@ -0,0 +1,18 @@
+---
+'@ag.ds-next/react': minor
+---
+
+## Box
+
+`Flex` and `Stack` have been moved into their own entrypoints. 
+
+These components will continue to work out of the the `@ag.ds-next/react/box` entrypoint, but this usage has been marked as deprecated and will be removed in the next major release.
+
+To upgrade, simply update the import when using these components.
```

Comment:
This is not _important_ but I like to avoid words like "simply" and "just" where possible.